### DISCUSSION TITLE
has anyone cracked the more than one person in a prompt?

### DISCUSSION THREAD

**Discussion by lgwjames** [lllyasviel/Fooocus#2277] [Q&A]
been trying to get people in the image with my ai model, and seems very hit and miss

like if i put say my ai model __jessicaL__ , standing with a asian female, wearing  what ever , i always get 2 Asian girls

**Comment by mashb1t** [comment]
curremtly this is not reliably possible, you should switch to an interative approach with inpainting

**Comment by lgwjames** [comment]
trying a lora people count slider, so will report back
